src/NUnitFramework/tests/Constraints/DelayedConstraintTests.cs

```diff
@@ -103,6 +103,7 @@ public void SimpleTest()
         }
 
         [Test]
+        [Explicit("This test takes over a minute to run")]
```

Comment:
This test slipped in recently in a pull requests. This one test causes over a minute to run. Multiplied by the six platforms made running the tests painful. We have two other tests that test canceling and stopping test runs that add 20 seconds per platform. I would like to make those explicit also, but wanted to get @nunit/core-team's opinion first. I think that our tests taking a couple of minutes to run is too long. Do you agree?